src/v2/view-builder/views/PollView.js

```diff
@@ -43,17 +47,34 @@ const Body = BaseForm.extend(Object.assign(
 
     remove() {
       BaseForm.prototype.remove.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
     },
 
     triggerAfterError() {
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
+      this.stopCountDown();
       this.$el.find('.o-form-fieldset-container').empty();
     },
-  },
 
-  polling,
+    startCountDown(selector , interval) {
```

Comment:
devs can't reuse it if they are not using PollView.. what's the reason to move it ?

Reply:
The count down was mostly for this one view only. no other view shows the countdown. I moved it per @haishengwu-okta's suggestion coz it made sense to remove any confusion in the polling.js

Reply:
yeah, the count down is only meant for PollView. We can move to polling.js until it's going to be reused by other cases.